cvrf2csaf/cvrf2csaf.py

```diff
@@ -175,6 +198,10 @@ def main():
     h = DocumentHandler(config)
     final_csaf = h.convert_file(path=config.get('input_file'))
 
+    if not h.validate_output_against_schema(config.get('output_file')):
+        # TODO: If --force given, the output should not be written when CSAF not valid according to schema
```

Comment:
It is exactly the other way around: If `--force` **is not** given, the output should not be written when CSAF not valid according to schema (and the business tests).

Reply:
ah sure, just a comment mistake, thx